tools/github/print-runtime-release-issue.ts

```diff
@@ -32,30 +32,39 @@ async function main() {
     `
 ## Release
 - [ ] Check all proxy types.
-- [ ] Re-run all extrinsics/hooks benchmarks.
-- [ ] Branch from master and create branch \`perm-runtime-${newVersion}\`.
-- [ ] Tag \`perm-${newVersion}\` with runtime-${newVersion} and push to github
-- [ ] Tag \`perm-runtime-${newVersion}\` with runtime-${newVersion}-templates and push to github
-- [ ] Tag \`perm-runtime-${newVersion}\` with runtime-${newVersion}-starlight and push to github
-- [ ] NOTE: if this is a hotfix to one of the runtimes, branch from runtime-${previousVersion} version
-and create perm-runtime-${newVersion}-templates, perm-runtime-${newVersion}-starlight or perm-runtime-${newVersion}-tanssi
-depending on whether the hotfix is for templates, starlight or tanssi-parachain. Then tag accordingly
-- [ ] Start the github action Publish Runtime Draft
-with runtime-${previousVersion} => runtime-${newVersion}
-  - \`gh workflow run "Publish Runtime Draft" -r 'master' ` +
-    `-f from=runtime-${previousVersion} -f to=runtime-${newVersion} -f chains=tanssi-only\`
+
+### Tanssi-para
+- [ ] Branch from master and create branch \`perm-runtime-${newVersion}-para\`
+- [ ] Tag \`perm-runtime-${newVersion}-para\` with runtime-${newVersion}-para and push to github
+- [ ] NOTE: if this is a hotfix to one of the runtimes, branch from runtime-${previousVersion}-para version
+and create perm-runtime-${newVersion}-para
 - [ ] Start the github action Publish Runtime Draft
-with runtime-${previousVersion} => runtime-${newVersion}
-  - \`gh workflow run "Publish Runtime Draft" -r 'master' ` +
-    `-f from=runtime-${previousVersion} -f to=runtime-${newVersion} -f chains=templates-only\`
+with runtime-${previousVersion}-para => runtime-${newVersion}-para orchestrator-para-only
+- [ ] Review the generated Draft and clean a bit the messages if needed (keep it draft)
+- [ ] Upgrade stagebox
```

Comment:
Can we extend it a bit? What steps included.

Reply:
in tanssi-networks? ok

Reply:
ah no this is runtime so no tanssi-networks. I updated with the extrinsic that needs to be called